modules/src/main/java/org/archive/modules/fetcher/AbstractCookieStore.java

```diff
@@ -221,7 +221,21 @@ protected LimitedCookieStoreFacade(List<Cookie> cookies) {
 
         @Override
         public boolean clearExpired(Date date) {
-            throw new RuntimeException("not implemented");
+            int expiredCount = 0;
+            for( Cookie c : cookies) {
+                boolean expired = AbstractCookieStore.this.expireCookie(c, date);
+                if( expired ) {
+                    logger.fine("Expired cookie: " + c + " for date: " + date);
+                    expiredCount++;
+                }
+            }
+            // Replace the cookie list, if any expired:
```

Comment:
I think this comment might be a stray comment leftover from the earlier commit? I don't think we replace the list any more?

Reply:
Yep, earlier iteration under there. I'll drop the comment.